src/sv-pipeline/scripts/preprocess_bed_for_vapor.py

```diff
@@ -78,9 +78,9 @@ def reformat(bed_in, bed_out, contig, sample_to_extract):
                                  fields[columns["name"]], svtype_write]) + "\n")
 
     if not found_contig:
-        raise ValueError(f"Could not find any records for contig {contig} in the provided BED file")
+        logging.warning(f"Could not find any records for contig {contig} in the provided BED file")
```

Comment:
Without making it too complicated, I wonder if it would be more useful to add an optional `--sex=male` flag to the argument list and throw the exception w.r.t to the value of this flag.

Reply:
This is a possible improvement but I'm going to leave it this way for now. We sometimes have unassigned sex or may want to run on a subset of contigs, which could make this more of a hindrance than a help.